.azure-pipelines/build.yml

```diff
@@ -57,8 +57,8 @@ jobs:
       ${{ else }}:
         artifact: sonic-swss-common.${{ parameters.arch }}
       runVersion: 'specific'
-      runId: 128696
-      runBranch: 'refs/heads/master'
+      runId: 146938
```

Comment:
Might as well use the latest build here. 201911 doesn't have the dependency issues.

Change `runVersion` to `latestFromBranch`, and remove `runId`.